Makefile

```diff
@@ -20,11 +20,11 @@
 #==================================================================================================
 
 
-CPP                =
+CPP                = 
 PYTHON             = python
 COMPILER_MODE      = DEBUG
 PRECISION          = DOUBLE
-OPENMP             = 1
+OPENMP             = 0
```

Comment:
I think OpenMP should be switched on by default. MPI is useful only if you are running on multiple machines, so it makes sense is not there by default. But really I can't see a reason why you wouldn't want openMP. Fixed this in 33c1ec6